.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,78 @@
+name: Bug report
+title: "Bug Report:"
+# Inspired by the [GoDot](https://github.com/godotengine/godot/blob/master/.github/ISSUE_TEMPLATE/bug_report.yml) bug report template.
+description: Report an issue to help us improve.
+labels: needs-triage, needs-sig, kind/bug
+body:
+
+- type: markdown
+  attributes:
+    value: |
+      - Read our [CONTRIBUTING.md guide](https://github.com/o3de/community/blob/main/CONTRIBUTING.md) on reporting bugs.
+      - Write a descriptive issue title above.
```

Comment:
nit: Maybe 'concise' issue title here might be better (the descriptive part can go in the description box)